src/main/java/uk/gov/hmcts/ccd/endpoint/std/CaseDetailsEndpoint.java

```diff
@@ -506,4 +533,34 @@ private MetaData createMetadata(String jurisdictionId, String caseTypeId, Map<St
 
         return metadata;
     }
+
+    private void validateSearchRequest(String searchRequest) throws IOException {
+        validateSearchRequestContainsQuery(searchRequest);
+        rejectBlackListedQuery(searchRequest);
+    }
+
+    private Optional<Map> getQuery(String searchRequest) throws IOException {
+        ObjectMapper mapper = new ObjectMapper();
```

Comment:
Can the ObjectMapper be injected as a bean rather than creating new instance for each query? As implemented in ElasticSearchCaseDetailsSearchOperation?

Reply:
It could @gbenadikar . I didn't do that because the object mapper injected in ElasticSearchCaseDetailsSearchOperation is the one that is supposed to be used to deserialise CaseDetails. And it has particular configuration specifically for that, which could change in the future. It should not be used anywhere a ObjectMapper is needed. 

Let's do this, I will configure a second general purpose ObjectMapper and will inject that here, cheers

Reply:
@gbenadikar I've tried that but unfortunately I'm getting the following exception: 
```java.lang.NoSuchMethodError: com.fasterxml.jackson.databind.DeserializationContext.wrongTokenException(Lcom/fasterxml/jackson/core/JsonParser;Ljava/lang/Class;Lcom/fasterxml/jackson/core/JsonToken;Ljava/lang/String;)Lcom/fasterxml/jackson/databind/JsonMappingException;```

I can't dedicate more time to investigate that at the moment. 
Let's ignore this change if you don't mind since it's a minor.

Reply:
no worries